### DISCUSSION TITLE
<Connecting Hydrophones to Jetson Nano>

### DISCUSSION THREAD

**Discussion by ParvRaval2336** [community/community#62144] [Discussions]
### Select Topic Area

Question

### Body

I want to connect 3 hydrophones to jetson nano and get amplitude and frequency from them. Is it possible to connect directly three hydrophones ?......If not what other components should i use?

**Comment by queenofcorgis** [comment]
Hi @ParvRaval2336 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on hydrophones. It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community on Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
